README.md

```diff
@@ -43,6 +43,57 @@ Miti.to_bs("2022/10/12").to_s(separator: "/") # separator can be [" ", "/"]
 Miti.to_ad("2079/06/26")
   #=> #<Date: 2022-10-12 ((2459865j,0s,0n),+0s,2299161j)>
 ```
+
+## MITI CLI
```

Comment:
Since we have CLI as well as date conversion as major highlights, it might be better if we  might want to list out content with sub branching.

Reply:
Duly Noted

Reply:
Duly Noted!